TWT/apps/challenges/models/challenge.py

```diff
@@ -70,3 +70,6 @@ def __str__(self):
     @property
     def active(self):
         return self.posted and not self.ended
+
+    def get_video_embed_link(self) -> str:
+        return f"https://youtube.com/embed/{self.youtube_video_link.split('=')[1]}"
```

Comment:
What for `youtu.be/...` links?

Reply:
@takos22 its for the admin page :p may be we can skip that part, but sure we can resolve that, but for youtu.be links the regex is also going to fail in the model so we would also have to change that @KrishnaKanth1729

Reply:
The regex accepts `youtu.be` links, that's why I asked about it

https://github.com/Tech-With-Tim/twtcodejam.net/blob/dev/TWT/apps/challenges/models/challenge.py#L65

Reply:
oh, yea then thats an issue

Reply:
ok so u want me to remove the youtu.be part?